include/picongpu/particles/externalBeam/beam/beamProfiles/GaussianProfile.hpp

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2020-2023 Pawel Ordyna
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+namespace picongpu
+{
+    namespace particles
+    {
+        namespace externalBeam
+        {
+            namespace beam
+            {
+                namespace beamProfiles
+                {
+                    template<typename T_ParamClass>
+                    struct GaussianProfile : public T_ParamClass
+                    {
+                        using ParamClass = T_ParamClass;
+
+                        static HDINLINE float_X getFactor(float_X const& x, float_X const& y)
+                        {
+                            constexpr float_X s_x = ParamClass::sigmaX_SI / UNIT_LENGTH;
+                            constexpr float_X s_y = ParamClass::sigmaY_SI / UNIT_LENGTH;
```

Comment:
Maybe `s_x` and `s_y` should be checked by a static_assert to avoid `zero` and a devision by zero in the next lines.